test/document_stores/test_opensearch.py

```diff
@@ -335,6 +335,92 @@ def test_query_by_embedding_excluded_meta_data_return_embedding_false(self, mock
         _, kwargs = mocked_document_store.client.search.call_args
         assert kwargs["body"]["_source"] == {"excludes": ["foo", "embedding"]}
 
+    @pytest.mark.unit
```

Comment:
Let's move all these tests in `SearchEngineDocumentStoreTestAbstract`, so we can reuse them for ES and OS

Reply:
done